bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/resources/OH-INF/thing/daikinmadoka.xml

```diff
@@ -52,6 +57,11 @@
 		<label>Unit Power Status</label>
 	</channel-type>
 
+	<channel-type id="brc1h_cleanFilter">
+		<item-type>Switch</item-type>
+		<label>Clean Filter Indicator</label>
+	</channel-type>
```

Comment:
Is this supposed to be a writable channel?

Reply:
Yes, writing "OFF" cleans the notification on the thermostat and restarts the clean-filter timer.